tarball distclean broken

Starting with pdns-3.3.1.tar.gz, and ./configure, running make distclean gives:

```
Making distclean in pdns/ext/rapidjson
make[1]: Entering directory `/home/ch/tmp/pdns-3.3.1/pdns/ext/rapidjson'
make[1]: *** No rule to make target `distclean'.  Stop.
make[1]: Leaving directory `/home/ch/tmp/pdns-3.3.1/pdns/ext/rapidjson'
make: *** [distclean-recursive] Error 1
```

After copying pdns/ext/rapidjson/Makefile from git:

```
Making distclean in pdns/ext/rapidjson
Making distclean in pdns
[...]
make[2]: Leaving directory `/home/ch/tmp/pdns-3.3.1/pdns/backends'
Making distclean in .
make[2]: Entering directory `/home/ch/tmp/pdns-3.3.1/pdns'
[...]
Makefile:1311: backends/bind/.deps/bindbackend2.Po: No such file or directory
Makefile:1312: backends/bind/.deps/binddnssec.Po: No such file or directory
Makefile:1313: backends/bind/.deps/bindlexer.Po: No such file or directory
Makefile:1314: backends/bind/.deps/bindparser.Po: No such file or directory
make[2]: *** No rule to make target `backends/bind/.deps/bindparser.Po'.  Stop.
make[2]: Leaving directory `/home/ch/tmp/pdns-3.3.1/pdns'
make[1]: *** [distclean-recursive] Error 1
make[1]: Leaving directory `/home/ch/tmp/pdns-3.3.1/pdns'
make: *** [distclean-recursive] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -10,7 +10,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_AUX_DIR([build-aux])
 
-AM_INIT_AUTOMAKE([foreign tar-ustar -Wno-portability subdir-objects])
+AM_INIT_AUTOMAKE([foreign tar-ustar -Wno-portability subdir-objects no-dependencies])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 AC_CANONICAL_HOST

